release-notes/unreleased/IGAPP-1136-fix-overlapping-pois-hiding-each-other.yml

```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-1136
+show_in_stores: false
+platforms:
+  - web
+  - android
+  - ios
+en: Fix overlapping POIs hiding each other
```

Comment:
pls add german translation for play store

Reply:
Only necessary if `show_in_stores` is set to true. But I think it should be set to true so I agree with @f1sh1918 .

Reply:
oh sorry didn't see that.